timesketch/api/v1/resources/analysis.py

```diff
@@ -62,6 +63,47 @@ def get(self, sketch_id, timeline_id):
         return self.to_json(analysis_history)
 
 
+class AnalyzerSessionActiveListResource(resources.ResourceMixin, Resource):
+    """Resource to get analyzer session."""
+
+    @login_required
+    def get(self, sketch_id):
+        """Handles GET request to the resource.
+
+        Returns:
+            A analyzer session in JSON (instance of flask.wrappers.Response)
+        """
+        sketch = Sketch.query.get_with_acl(sketch_id)
+
+        if not sketch:
+            abort(
+                HTTP_STATUS_CODE_NOT_FOUND, 'No sketch found with this ID.')
+
+        if not sketch.has_permission(current_user, 'read'):
+            abort(
+                HTTP_STATUS_CODE_FORBIDDEN,
+                'User does not have read access to sketch')
+
+        active_sessions = sketch.get_active_analysis_sessions()
+
+        counter = collections.Counter(PENDING=0, STARTED=0, ERROR=0, DONE=0)
+        session_ids = set()
+        for session in active_sessions:
```

Comment:
tiny nit... can you define `active_sessions` just before it's use in the for loop, makes it easier to read